manager/orchestrator/taskreaper/task_reaper.go

```diff
@@ -254,8 +219,6 @@ func (tr *TaskReaper) tick() {
 // Stop stops the TaskReaper and waits for the main loop to exit.
 func (tr *TaskReaper) Stop() {
 	tr.cancelWatch()
-	// TODO(dperny) calling stop on the task reaper twice will cause a panic
```

Comment:
Not that it matters (this is just a comment), but the cherry pick commit doesn't modify this comment: https://github.com/docker/swarmkit/commit/5582db0bfeb6005e24578dc75815ccd8fc96be02

Let's keep consistency so that things don't bother us later.

Reply:
Ah okay, then ignore my comment.